apps/heft/src/schemas/node-service.schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Node Service Configuration",
+  "description": "Configures the Node Service plugin for Heft",
+  "type": "object",
+
+  "additionalProperties": false,
+
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+
+    "extends": {
+      "description": "Optionally specifies another JSON config file that this file extends from.  This provides a way for standard settings to be shared across multiple projects.",
+      "type": "string"
+    },
+
+    "enabled": {
+      "description": "Set this to \"false\" to disable the node service plugin.",
+      "type": "boolean"
+    },
+
+    "commandName": {
+      "description": "Specifies the name of a \"scripts\" command from the project's package.json file. When \"heft start\" is invoked, it will use this shell command to launch the service process.",
+      "type": "string"
+    },
+
+    "ignoreMissingScript": {
+      "description": "If true, then an error is reported if the \"scripts\" command is not found in the project's package.json.  If false, then no action will be taken.",
+      "type": "boolean"
```

Comment:
If the param is called `ignore` than error should be reported on `false`, not on `true`, I think?

Reply:
Good catch, thanks!